modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/NotificationQueueEntryStatusDataRenderer.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+
+export default function ObjectActionStatusDataRenderer({value}: IProps) {
+	return value ? (
+		<strong className="label label-success">sent</strong>
```

Comment:
@p-albuquerque Shouldn't we use key languages for it? If we change the portal language?

Reply:
@p-albuquerque,we can also make this component more generic by passing the `success` and `danger` statuses as a component property, which is better if we need to use the same component with different labels, what do you think?